applications/operator_sdk/osdk-leader-election.adoc

```diff
@@ -0,0 +1,41 @@
+[id="osdk-leader-election"]
+= Configuring leader election
+include::modules/common-attributes.adoc[]
+:context: osdk-leader-election
+
+toc::[]
+
+During the lifecycle of an Operator, it is possible that there may be more than
+one instance running at any given time, for example when rolling out an upgrade
+for the Operator. In such a scenario, it is necessary to avoid contention
+between multiple Operator instances via leader election. This ensures only one
```

Comment:
We've been getting pushback on using "via" (IBM style guide doesn't like it)

Reply:
OOH yes, already changed some other "via"s, missed this one, thx.